lib/node_modules/@stdlib/math/base/special/gammasgn/README.md

```diff
@@ -0,0 +1,127 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2022 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# gammasgn
+
+> Sign of the [gamma function][gamma-function].
+
+Defined as,
+
+<section class="intro">
+
+<!-- <equation class="equation" label="eq:gamma_sign_function" align="center" raw="gammasgn ( x ) = \begin{cases} 1 & \textrm{if}\ \Gamma > 1 \\ -1 & \textrm{if}\ \Gamma < 1 \\ 0 & \textrm{otherwise}\ \end{cases}" alt="Gamma sign function"> -->
+
+<div class="equation" align="center" data-raw-text="gammasgn ( x ) = \begin{cases} 1 &amp; \textrm{if}\ \Gamma &gt; 1 \\ -1 &amp; \textrm{if}\ \Gamma &lt; 1 \\ 0 &amp; \textrm{otherwise}\ \end{cases}" data-equation="eq:gamma_sign_function">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@9ab12e34e11c18184aaed8f140943fe5dcf30d39/lib/node_modules/@stdlib/math/base/special/gammasgn/docs/img/equation_gamma_sign_function.svg" alt="Gamma sign function">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+The [gamma function][gamma-function] can be computed as the product of `gammasgn` and `gammaln`.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var gammasgn = require( '@stdlib/math/base/special/gammasgn' );
+```
+
+#### gammasgn( x )
+
+Returns the sign of the [gamma function][gamma-function] evaluated in the given `x` value.
+
+```javascript
+var v = gammasgn( 1.0 );
+// returns 1.0
+
+v = gammasgn( -2.5 );
+// returns -1.0
+
+v = gammasgn( 0.0 );
+// returns 0.0
+
+v = gammasgn( NaN );
+// returns NaN
+```
+
+</section>
+
+<!-- /.usage -->
+
```

Comment:
We should add a notes section here to explain why the `gammasgn` function can return `0.0` and why that makes sense.

Namely, the gamma function is not defined for negative integer values (i.e., `gamma(x) === NaN` when `x` is a negative integer). The natural logarithm of the gamma function is defined for negative integer values (e.g., `gammaln(x) === Infinity` when `x` is a negative integer). Accordingly, in order for the equality `gamma(x) === gammasgn(x) * exp(gammaln(x))` to hold (i.e., return `NaN`), `gammasgn` needs to either return `NaN` or `0`. By convention, this function returns `0`.